pkg/nuclide-ocaml/lib/main.js

```diff
@@ -41,6 +42,16 @@ export function provideLinter(): LinterProvider {
   return MerlinLinterProvider;
 }
 
+export function provideOutlines(): OutlineProvider {
+  const {getOutline} = require('./OutlineProvider');
```

Comment:
You can just `import {getOutline} from './OutlineProvider';` at the top of the file. We have a babel transformation in place that will not actually `require` the file until `getOutline` is referenced so there is no performance benefit to lazy-requiring explicitly.